src/test/run-make-fulldeps/instrument-coverage/expected_coverage_summary.json

```diff
@@ -0,0 +1,163 @@
+{
+  "data": [
+    {
+      "files": [
+        {
+          "expansions": [],
+          "filename": "/usr/local/google/home/richkadel/rust/src/test/run-make-fulldeps/instrument-coverage/main.rs",
+          "segments": [
+            [
+              1,
+              25,
+              1,
+              true,
+              true
+            ],
+            [
+              3,
+              2,
+              0,
+              false,
+              false
+            ],
+            [
+              5,
+              29,
+              0,
+              true,
+              true
+            ],
+            [
+              7,
+              2,
+              0,
+              false,
+              false
+            ],
+            [
+              9,
+              11,
+              1,
+              true,
+              true
+            ],
+            [
+              18,
+              2,
+              0,
+              false,
+              false
+            ]
+          ],
+          "summary": {
+            "functions": {
+              "count": 3,
+              "covered": 2,
+              "percent": 66.66666666666666
+            },
+            "instantiations": {
+              "count": 3,
+              "covered": 2,
+              "percent": 66.66666666666666
+            },
+            "lines": {
+              "count": 16,
+              "covered": 13,
+              "percent": 81.25
+            },
+            "regions": {
+              "count": 3,
+              "covered": 2,
+              "notcovered": 1,
+              "percent": 66.66666666666666
+            }
+          }
+        }
+      ],
+      "functions": [
+        {
+          "count": 1,
+          "filenames": [
+            "/usr/local/google/home/richkadel/rust/src/test/run-make-fulldeps/instrument-coverage/main.rs"
+          ],
+          "name": "_ZN4main14will_be_called17h29a91959e2e86227E",
```

Comment:
Making a note to investigate: I don't remember what exactly goes into these hashes, but am slightly worried it could change between different compiler versions.

Reply:
I thought the newer mangler was supposed to overcome that problem?

Reply:
Maybe, but it still isn't enabled by default yet.

Reply:
I'm no enabling `v0` name mangling automatically, when `-Zinstrument-coverage` is enabled. This makes the coverage output much better too, when running `llvm-cov` with `-Xdemangler=rust-demangler`.